src/domain/synthetics/markets/useClaimableFundingDataRequest.ts

```diff
@@ -0,0 +1,97 @@
+import DataStore from "abis/DataStore.json";
+import { getContract } from "config/contracts";
+import { CLAIMABLE_FUNDING_AMOUNT } from "config/dataStore";
+import { useMulticall } from "lib/multicall";
+import { hashDataMap } from "lib/multicall/hashDataMap";
+import { getByKey } from "lib/objects";
+import { useMemo } from "react";
+import { ClaimableFundingData, getMarketDivisor, useMarkets } from ".";
+import { FREQUENT_MULTICALL_REFRESH_INTERVAL } from "lib/timeConstants";
+import useWallet from "lib/wallets/useWallet";
+
+export function useClaimableFundingDataRequest(chainId: number) {
+  const { account } = useWallet();
+  const { marketsAddresses, marketsData } = useMarkets(chainId);
+
+  const {
+    data: claimableFundingData,
+    isLoading,
+    error,
+  } = useMulticall(chainId, "useClaimableFundingDataRequest", {
+    key: account && marketsAddresses?.length ? [account, marketsAddresses] : null,
```

Comment:
it should most likely be fine, but make sure, let's do `...marketAddresses` just in case?